datasets/objectenopenbareruimte/dataset.json

```diff
@@ -33,7 +33,11 @@
     {
       "id": "verhardingen",
       "type": "table",
-      "auth": "FP/MDW",
+      "auth": 
+        [
+        "FP/IASSET",
+        "FP/MDW"
```

Comment:
Waarom wil je dit? FP/MDW bevat alle medewerkers. Dus iedereen in FP/IASSET zit toch ook al in FP/MDW?

Reply:
Deze dataset is niet mag niet publiekelijk opengezet worden vandaar FB/MDW. De uitvoeringsapplicatie I-Asset is een applicatie die buiten de gemeentelijke netwerk zich bevindt. Deze applicatie moet  de data via de scope "FP/IASSET" kunnen laden. Vandaar een extra scope.    
  
Ik maak o.b.v. jou opmerking dat alleen de scope FP/IASSET voldoende zou zijn. Klopt deze aanname?